<launch>
  <node name="align_bag" type="align_bag.py" pkg="misc_bag_utils" output="screen">
    <param name="input_file_name" value="/home/icoderaven/raven_first_switching.bag" />
    <param name="imu_topic" value="/raven/imu" />

    <param name="align_odom" value="true" />
    <param name="num_odom" value="2" />
    <param name="odom_topic1" value="/raven/mocap/odom" />
    <param name="odom_topic2" value="/raven/svo/odom" />

    <param name="align_camera" value="false" />
    <param name="imu_camera_dt" value="" />
    <param name="left_camera_topic" value="" />
    <param name="right_camera_topic" value="" />
  </node>
</launch>
